app/lib/efile/az/az140.rb

```diff
@@ -55,12 +55,8 @@ def calculate
           set_line(:AZ140_LINE_62, -> { 0 })
           set_line(:AZ140_LINE_63, :calculate_line_63)
         end
-        if (line_or_zero(:AZ140_LINE_63) - line_or_zero(:AZ140_LINE_78)) >= 0
-          set_line(:AZ140_LINE_79, :calculate_line_79)
-        else
-          set_line(:AZ140_LINE_80, :calculate_line_80)
-        end
         set_line(:AZ140_LINE_79, :calculate_line_79)
+        set_line(:AZ140_LINE_80, :calculate_line_80)
```

Comment:
line 79 was always being set and line 80 never was because stuff is always at least 0

i tracked this down because the review page needed to display the refund or owed amount and it was always 0